tests/utils.go

```diff
@@ -4584,6 +4631,11 @@ func IsRunningOnKindInfra() bool {
 	return strings.HasPrefix(provider, "kind")
 }
 
+func IsRunningOnKindInfraIPv6() bool {
+	provider := os.Getenv("KUBEVIRT_PROVIDER")
+	return strings.HasPrefix(provider, "kind-k8s-1.17.0-ipv6")
```

Comment:
@oshoval is it something specific for 1.17?

Reply:
its not specific, 
for the moment as we groomed with you and Alona and the team,
it was decided that indeed the change is better to exists, 
for future usage of kind that will need migration,
but will be done separately
https://github.com/kubevirt/kubevirtci/issues/353

Reply:
Yeah, I not against it :) 
Perhaps we shouldn't check for `1.17.0`, but rather check for prefix and suffix?

Reply:
kind*ipv6 if i understood right
hmm it means that in a case there will be other that have this pattern it would fail
unless the kubevirtci part would be added there as well.

also at the moment just this provider has the needed code in the kubevirtci,
the change there isnt hard and then the if will be indeed kind* only
so i believe its fine as its now imo ?

or i didnt understand and the reason is different ?

Reply:
glad to hear btw, thanks